.github/workflows/main.yml

```
@@ -0,0 +1,59 @@
+env:
+  RUBY_VERSION: 2.3.7
+  FLOW_TOKEN: testflow
+
+name: Code Quality
+
+on: [push]
+
+jobs:
+  rspec-test:
+    name: RSpec
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11.6
+        ports: ["5432:5432"]
+        env:
+          POSTGRES_USER: ${{ secrets.POSTGRES_USER }}
+          POSTGRES_PASSWORD: ${{ secrets.POSTGRES_PASSWORD }}
+          POSTGRES_DB: ${{ secrets.POSTGRES_DB }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v1  
+        with: 
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}  
+          path: vendor/bundle
+      - uses: ruby/setup-ruby@master
+        id: ruby
+        with:
+          ruby-version: ${{ env.RUBY_VERSION }}
+      - name: Install postgres client
+        run: sudo apt-get install libpq-dev
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install libcurl3-dev
+          gem install bundler -v 1.10.6
+          bundler install
+      - name: Create database
+        run: |
+          bundler exec rake db:schema:load RAILS_ENV=test
+      - name: Test & Upload coverage results    
+        uses: paambaati/codeclimate-action@v2.7.5
+        env:
+          CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+        with:
+          coverageCommand: bundler exec rake REPORT_COVERAGE=true
+          debug: true
+  code-check:
+    name:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
```

I think we might try to use v2 of checkout - we're already using it above, in line 22

```suggestion
    - uses: actions/checkout@v2
```

---

Thanks Aurel! I will make these changes you mentioned in the next push!! I honestly simply went with the same configuration we already have in mejuri-web... but if it works ok then we can also move these changes there !!